src/helper_sql.py

```diff
@@ -33,6 +36,15 @@
    initialization is done)"""
 
 
+def executeCF(conn):
```

Comment:
docstring missing

Reply:
Also I think this would be better inside `class_sqlThread`. This function doesn't need to be called from other places.